core/invoker/src/main/resources/application.conf

```diff
@@ -71,6 +71,7 @@ whisk {
     prewarm-promotion: false # if true, action can take prewarm container which has bigger memory
     memory-sync-interval: 1 second # period to sync memory info to etcd
     batch-deletion-size: 10 # batch size for removing containers when disable invoker, too big value may cause docker/k8s overload
+    # user-cpus: 1 optional setting to limit total allocated cpus for all action containers, each container will get a fraction of this proportional to its allocated memory
```

Comment:
Is this supposed to be the cpu core of an invoker machine?

Reply:
For most cases yes, but you could set this lower than the actual number of cores the invoker machine has if you wanted to reserve some amount of cpu for the invoker/docker processes.

Reply:
```suggestion
    # user-cpus: 1 # optional setting to specify the total allocatable cpus for all action containers, each container will get a fraction of this proportional to its allocated memory to limit the cpu
```